pyproject.toml

```diff
@@ -1381,83 +1387,1472 @@ combine-as-imports = true
+"airflow/utils/net.py" = ["D100"]
+"airflow/utils/operator_helpers.py" = ["D100", "D102"]
+"airflow/utils/operator_resources.py" = ["D100", "D102", "D105"]
+"airflow/utils/orm_event_handlers.py" = ["D100"]
+"airflow/utils/providers_configuration_loader.py" = ["D100"]
+"airflow/utils/pydantic.py" = ["D100", "D103"]
+"airflow/utils/python_virtualenv.py" = ["D103"]
+"airflow/utils/retries.py" = ["D100"]
+"airflow/utils/scheduler_health.py" = ["D100", "D102"]
+"airflow/utils/sensor_helper.py" = ["D100"]
+"airflow/utils/serve_logs.py" = ["D102", "D103"]
+"airflow/utils/session.py" = ["D100"]
+"airflow/utils/setup_teardown.py" = ["D100", "D102"]
+"airflow/utils/singleton.py" = ["D100", "D102"]
+"airflow/utils/sqlalchemy.py" = ["D100", "D102", "D105"]
+"airflow/utils/state.py" = ["D100", "D105"]
+"airflow/utils/task_group.py" = ["D102", "D105"]
+"airflow/utils/task_instance_session.py" = ["D100", "D103"]
+"airflow/utils/template.py" = ["D100"]
+"airflow/utils/timeout.py" = ["D100", "D105"]
+"airflow/utils/timezone.py" = ["D100"]
+"airflow/utils/trigger_rule.py" = ["D100", "D105"]
+"airflow/utils/types.py" = ["D100", "D102", "D105"]
+"airflow/utils/weight_rule.py" = ["D100", "D105"]
+"airflow/utils/xcom.py" = ["D100"]
+"airflow/version.py" = ["D100"]
+"airflow/www/__init__.py" = ["D104"]
+"airflow/www/api/__init__.py" = ["D104"]
+"airflow/www/api/experimental/__init__.py" = ["D104"]
+"airflow/www/api/experimental/endpoints.py" = ["D100"]
+"airflow/www/app.py" = ["D100"]
+"airflow/www/auth.py" = ["D100", "D103"]
+"airflow/www/blueprints.py" = ["D100"]
+"airflow/www/decorators.py" = ["D100"]
+"airflow/www/extensions/__init__.py" = ["D104"]
+"airflow/www/extensions/init_appbuilder.py" = ["D100", "D102"]
+"airflow/www/extensions/init_appbuilder_links.py" = ["D100"]
+"airflow/www/extensions/init_auth_manager.py" = ["D100"]
+"airflow/www/extensions/init_cache.py" = ["D100", "D103"]
+"airflow/www/extensions/init_dagbag.py" = ["D100"]
+"airflow/www/extensions/init_jinja_globals.py" = ["D100"]
+"airflow/www/extensions/init_manifest_files.py" = ["D100"]
+"airflow/www/extensions/init_robots.py" = ["D100"]
+"airflow/www/extensions/init_security.py" = ["D100", "D103"]
+"airflow/www/extensions/init_session.py" = ["D100"]
+"airflow/www/extensions/init_views.py" = ["D100"]
+"airflow/www/extensions/init_wsgi_middlewares.py" = ["D100"]
+"airflow/www/fab_security/__init__.py" = ["D104"]
+"airflow/www/fab_security/manager.py" = ["D100"]
+"airflow/www/forms.py" = ["D100", "D102"]
+"airflow/www/gunicorn_config.py" = ["D100", "D103"]
+"airflow/www/security.py" = ["D100"]
+"airflow/www/security_appless.py" = ["D100"]
+"airflow/www/security_manager.py" = ["D100", "D102"]
+"airflow/www/session.py" = ["D100"]
+"airflow/www/utils.py" = ["D100", "D102", "D103"]
+"airflow/www/validators.py" = ["D100", "D102"]
+"airflow/www/views.py" = ["D100", "D102", "D103"]
+"airflow/www/widgets.py" = ["D100", "D102"]
+"clients/python/test_python_client.py" = ["D100"]
```

Comment:
I think `clients/*` could be excluded from D checks, this one not included into the distribution of airflow or providers